src/components/CommentDisplay.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { fetchReviewsComments, fetchSingleReview } from "./api-requests";
+
+const CommentDisplay = ({review_id}) => {
+    const [loading, setLoading] = useState(true)
+    const [comments, setComments] = useState([])
+    
+    useEffect(() => {
+        setLoading(true)
+        fetchReviewsComments(review_id).then((commentData) => {
+            setLoading(false)
+            return setComments(commentData.comments)
+        })
+    }, [])
+
+    return (
```

Comment:
you need to consider the loading state here, you're always returning your comments